src/hardikBhatia/Test4.java

```diff
@@ -0,0 +1,50 @@
+package hardikBhatia;
+
+public class Test4 {
+	//Program 1 : Write a method to return sum of all the digits present in each elements from given
+	int getSumOfDigit(String name) {
+		int sum = 0;
+		for(int j = 0; j < name.length(); j++) {
+			if(Character.isDigit(name.charAt(j))) {
+				int numeric = Character.getNumericValue(name.charAt(j));
+				sum += numeric;
+			}
+		}
+		return sum;
+	}
+	void getSumOfDigitInString(String[] name) {
+		int count = 0;
+		for(int index = 0; index < name.length; index++) {
+			String currentName = name[index];
+			System.out.println(currentName + "---" + getSumOfDigit(currentName));
+			count++;
+		}
+	}
+	// Program 2 : Return count of visible trees. 11minit
+	int checkHowMuchTreesVisible(int[] treesHight){
+		int countOftreeVisible = 0;
```

Comment:
countOftreeVisible should be started from 1, as the first tree always visible in the row